src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -227,32 +228,32 @@ public boolean isDependentArgumentAllowed(final Class<?> predecessorClass) {
     @Override
     public void validateArguments() {
```

Comment:
For example, if a plugin requires one of two `ReadFilter` to be applied, this method will be useful to check if both of them are disabled (or in case one or the other should be applied, check if they are mutually exclusive).

Another example is a constraint in some `ReadFilter` parameters, that are not included in the original filter. For instance, if `ReadLenghtReadFilter` should have always a minimum length of 0.

I think that this will be easier at some point with Barclay anyway if https://github.com/broadinstitute/barclay/issues/31 is implemented (the later comment about the `Validatable` interface).